src/silo_api/info_handler.cpp

```diff
@@ -88,6 +88,8 @@ void InfoHandler::get(
 ) {
    const auto request_parameter = getQueryParameter(request);
 
+   response.set("data-version", database.getDataVersion().toString());
```

Comment:
I think headers are more often than not in upper case.
```suggestion
   response.set("Data-Version", database.getDataVersion().toString());
```

Reply:
As discussed, the data version is parsed in lower case in the e2e tests. Thus, we keep it lower case in the beginning. Additionally, it does not seem to have a real convention on the header case style.